umap/static/umap/js/umap.js

```diff
@@ -1960,17 +1940,17 @@ L.U.Map.include({
   },
 
   openExternalRouting: function (e) {
-    const url = this.options.urls.routing
-    if (url) {
-      const params = {
+    let url = this.urls.get(
+      'routing',
+      {
         lat: e.latlng.lat,
         lng: e.latlng.lng,
         locale: L.locale,
         zoom: this.getZoom(),
       }
-      window.open(L.Util.template(url, params))
-    }
-    return
+    );
+    if (url)
```

Comment:
We had a discussion about this simple `if` case with @davidbgk. We did not ended with a clear consensus, but I made this proposal:
- if the `if` is followed by a single instruction AND prettier keeps it on a single line, let's use a one liner without the curly bracket
- other wise put use curly brackets and carriage return

Reply:
Thanks for mentioning it :-)